packages/web-react/src/components/Card/CardTitle.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import classNames from 'classnames';
+import React, { ElementType } from 'react';
+import { useStyleProps } from '../../hooks';
+import { SpiritCardTitleProps } from '../../types';
+import { useCardStyleProps } from './useCardStyleProps';
+
+const defaultProps: Partial<SpiritCardTitleProps> = {
+  elementType: 'h4',
+  isHeading: true,
```

Comment:
**question (if-minor):** Shouldn't be this done via the `Heading` component? I think about the composability of our components. And this looks like we cannot use our Heading for Card titles.

Reply:
I was asking the same question and we should not, it should not behave like our heading.